lib/media/track/mediatrack.js

```diff
@@ -197,9 +197,18 @@ class MediaTrack extends Track {
       mediaStream.addTrack(mediaStreamTrack);
     }
 
+    const isEmptyVideoMediaStream = this.kind === 'video' && mediaStream.getTracks().length <= 0;
```

Comment:
@manjeshbhargav I am bit confused on when `mediaStream` will have no tracks here. Don't we `addTrack` just above this line? Do we ever call `_attach` with `mediaStreamTrack = null` ?

Reply:
@makarandp0 Yes, The mediaStreamTrack passed can be `null`, when the track is switched off in large rooms.